.circleci/docker/build.sh

```diff
@@ -53,11 +53,6 @@ case "$image" in
     DB=yes
     VISION=yes
     ;;
-  pytorch-linux-xenial-py3.5)
-    TRAVIS_PYTHON_VERSION=3.5
-    GCC_VERSION=7
-    # Do not install PROTOBUF, DB, and VISION as a test
```

Comment:
Looks like we're covered with these checks at pytorch-linux-xenial-py3.6-gcc7.2 which is good